gluten-delta/pom.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need to make change to include this on gluten-<backend_type>-bundle-spark?

Reply:
No. I hope these gluten-lakeformat modules can be used in both backends.